eofconvention.fsx

```diff
@@ -0,0 +1,23 @@
+open System.IO
+
+let EOFIsEOL(path: string) = 
+    try
+        use sr = new StreamReader (path)
+        let filetext = sr.ReadToEnd()
+        (filetext |> Seq.last = '\n')
+        
+    with ex ->
+        // printfn "%A" ex
+        true
+
+let EOFIsNotEOL(path: string) = not (EOFIsEOL path)
+
+let NotPrefix (value: string) (str: string) =
+    not (str.StartsWith(value))
+
+printfn "%A" (
+    Directory.GetFiles("./", "*.*", SearchOption.AllDirectories) 
```

Comment:
I think you don't need the slash, it should work with just "."

Reply:
Done